src/appmixer/google/gmail/ListAttachments/ListAttachments.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const emailCommons = require('../gmail-commons');
+
+module.exports = {
+    async receive(context) {
+        const { messageId } = context.messages.in.content;
+        const endpoint = `/users/me/messages/${messageId}`;
+        const messageDetails = await emailCommons.callEndpoint(context, endpoint, {
+            method: 'GET',
+            params: {
+                format: 'full'
+            }
+        });
+        const normalizedEmail = emailCommons.normalizeEmail(messageDetails.data);
+        context.log({ normalizedEmail });
```

Comment:
please use format like `{ stage: "<name of the stage, e.g nomalizing emails> <log message>" }` or remove the context.log